### DISCUSSION TITLE
After updating tailwind 3.0.7, I got this error. Can anyone help? 🙏🏽

### DISCUSSION THREAD

**Discussion by sarnielmorgia** [tailwindlabs/tailwindcss#6643] [Help] [answered]
error - ./node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[2].oneOf[8].use[1]!./node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[2].oneOf[8].use[2]!./node_modules/react-toastify/dist/ReactToastify.css
TypeError: Cannot read properties of undefined (reading '500')

**Comment by sarnielmorgia** [comment]
{
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "eslint \"**/*.{js,jsx,ts,tsx}\" --fix",
    "prepare": "husky install"
  },
  "dependencies": {
    "@headlessui/react": "^1.4.1",
    "@heroicons/react": "^1.0.4",
    "@react-google-maps/api": "^2.5.0",
    "@reduxjs/toolkit": "^1.6.2",
    "axios": "^0.22.0",
    "classnames": "^2.3.1",
    "date-fns": "^2.25.0",
    "deep-object-diff": "^1.1.0",
    "dompurify": "^2.3.3",
    "downshift": "^6.1.7",
    "draft-convert": "^2.1.12",
    "draft-js": "^0.11.7",
    "formik": "^2.2.9",
    "lodash.intersection": "^4.4.0",
    "lodash.isempty": "^4.4.0",
    "next": "^12.0.2",
    "nprogress": "^0.2.0",
    "react": "^17.0.2",
    "react-date-range": "^1.4.0",
    "react-dom": "^17.0.2",
    "react-phone-number-input": "^3.1.35",
    "react-places-autocomplete": "^7.3.0",
    "react-redux": "^7.2.5",
    "react-toastify": "^8.0.3",
    "react-tooltip": "^4.2.21",
    "recharts": "^2.1.6",
    "swiper": "^7.2.0",
    "use-position": "^1.0.0",
    "vault.js": "^1.0.3",
    "yup": "^0.32.9",
    "yup-phone": "^1.2.19"
  },
  "devDependencies": {
    "@tailwindcss/aspect-ratio": "^0.3.0",
    "@tailwindcss/forms": "^0.3.4",
    "@tailwindcss/line-clamp": "^0.2.1",
    "autoprefixer": "^10.2.6",
    "eslint": "^7.32.0",
    "eslint-config-next": "^11.1.2",
    "eslint-config-prettier": "^8.3.0",
    "husky": "^7.0.0",
    "lint-staged": "^11.2.0",
    "postcss": "^8.3.5",
    "postcss-import": "^14.0.2",
    "tailwindcss": "^3.0.7"
  }
}

**Comment by coopernewby** [comment, marked as answer]
I had the same issue as well, I think you need to update the tailwind forms plugin as well so it plays nicely with 3.0+

`npm install -D @tailwindcss/typography@latest \
@tailwindcss/forms@latest`

**Reply by simonswiss** [reply]
Hey, yep - this is correct, you need to upgrade the first party plugins (Typography, Forms, Line-clamp, Aspect ratio) to their latest versions:

https://tailwindcss.com/docs/upgrade-guide#official-plugins

**Comment by sarnielmorgia** [comment]
Hi everyone, yes it works well now. Thanks for the help.
